### DISCUSSION TITLE
What command I should run in terminal for starting neovim with agenda week or month view?

### DISCUSSION THREAD

**Discussion by VolkovIlia** [nvim-orgmode/orgmode#181] [Q&A] [answered]
Hello everyone. I use dwm and I want to bind opening nvim with agenda view. I did not find any instructions how to exec orgmode commands from nvim, just bindings in config. 
Can you give me instructions how to bind any commands or exec commands from nvim, especially for stating it with agenda view.

**Comment by kristijanhusak** [comment, marked as answer]
This command should open agenda view directly:

```
:lua require('orgmode').action('agenda.agenda')
```
